python-sdk/pyproject.toml

```diff
@@ -21,6 +21,7 @@ dependencies = [
     "attrs>=20.0",
     "pandas>=1.3.4,<2.0.0", # Pinning it to <2.0.0 to avoid breaking changes
     "pyarrow",
+    "pyodbc",
```

Comment:
Why do we need to make `pyodbc` as a required core dep here?

Do we need it for core or is this just needed for mssql?

Reply:
I added it just to test, removed it later